mongodb/src/main/scala/io/github/rafafrdz/criteria4s/mongodb/MongoDb.scala

```diff
@@ -0,0 +1,52 @@
+package io.github.rafafrdz.criteria4s.mongodb
+
+import io.github.rafafrdz.criteria4s.core.Conjuction.{AND, OR}
+import io.github.rafafrdz.criteria4s.core.{Column, CriteriaTag, Show}
+import io.github.rafafrdz.criteria4s.core.PredicateBinary._
+import io.github.rafafrdz.criteria4s.core.PredicateUnary._
+import io.github.rafafrdz.criteria4s.instances._
+
+trait MongoDb extends CriteriaTag
```

Comment:
```suggestion
trait MongoDB extends CriteriaTag
```

Reply:
Thanks for reacting! I really appreciate your comments :)

1. I agree. I guess it also doesn't hurt the scala codestyle as this naming is specific for MongoDB.
2. I like it. Thus we separate our interpreters from the algebra I think. Good idea!

So then, can you merge firstly your https://github.com/rafafrdz/criteria4s/pull/25 PR? Then I'll make the mentioned corrections and merge with your changes properly

Reply:
Perfect!

> So then, can you merge firstly your https://github.com/rafafrdz/criteria4s/pull/25 PR? Then I'll make the mentioned corrections and merge with your changes properly

approve me it (#25) please 🤖

Reply:
Ready😉